From ac3414201a9331317f0b3ffb1e33235fcff2fe77 Mon Sep 17 00:00:00 2001
From: Sean Rhodes <sean@starlabs.systems>
Date: Sun, 14 Jul 2024 20:28:00 +0100
Subject: [PATCH] Revert "hwdb: Added StarLabs StarLite position sensor
 mapping"

This reverts commit 1bd5db86f50feb2efd8e54e8156816d9727f6f2f.

The `kxcjk-1013` driver in Linux will parse the rotation matrix
from ACPI. This quirk is not specific enough to exist without
causing issues on different variations.

Signed-off-by: Sean Rhodes <sean@starlabs.systems>
(cherry picked from commit d350651c47da0377d835b926d6cabe292f10c745)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index 94f8aa5099..6abb6c5238 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -1025,13 +1025,6 @@ sensor:modalias:acpi:KIOX000A*:dmi:*:svnRCA:pnW101SA23T1:*
 sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvrSCH12i.WJ210Z.KtBJRCA03:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
 
-#########################################
-# StarLabs Systems
-#########################################
-# StarLite
-sensor:modalias:acpi:KIOX000A*:dmi:*:svnStarLabs:pnStarLite:*
- ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
-
 #########################################
 # Teclast
 #########################################
